changelog.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 
 2020-0x-xx • `full history <https://github.com/gorakhargosh/watchdog/compare/v0.10.2...master>`__
 
+- Ensure ObservedWatch.path is a string (`#655 <https://github.com/gorakhargosh/watchdog/pull/651>`_)
```

Comment:
```suggestion
- Ensure ``ObservedWatch.path`` is a string (`#651 <https://github.com/gorakhargosh/watchdog/pull/651>`_)
```
Let's wait for the tests to pass first. Then you can amend the commit and I will merge right after.